src/themes/stories/tokens.stories.tsx

```diff
@@ -20,6 +20,26 @@ const conversionToPixels = {
     'Pixel values are only used as a reference for Figma designs. We use rem as units.',
 };
 
+export const AspectRatios: StoryType = {
```

Comment:
If you're adding new token category - please add them to the storybook showcase so we can see the theme easily